shared/src/main/kotlin/org/javacs/kt/classpath/MavenClassPathResolver.kt

```diff
@@ -58,21 +59,34 @@ private fun mavenJarName(a: Artifact, source: Boolean) =
     if (source) "${a.artifact}-${a.version}-sources.jar"
     else "${a.artifact}-${a.version}.jar"
 
-private fun generateMavenDependencyList(pom: Path): Path {
-    val mavenOutput = Files.createTempFile("deps", ".txt")
-    val workingDirectory = pom.toAbsolutePath().parent.toFile()
-    val cmd = "$mvnCommand dependency:list -DincludeScope=test -DoutputFile=$mavenOutput"
-    LOG.info("Run {} in {}", cmd, workingDirectory)
-    val process = Runtime.getRuntime().exec(cmd, null, workingDirectory)
-
-    process.inputStream.bufferedReader().use { reader ->
+private fun readInputStream(process: Process, inputStream: InputStream) {
+    inputStream.bufferedReader().use { reader ->
         while (process.isAlive) {
             val line = reader.readLine()?.trim() ?: break
             if (line.isNotEmpty() && !line.startsWith("Progress")) {
                 LOG.info("Maven: {}", line)
             }
         }
     }
+}
```

Comment:
Logging dependencies directly from Maven's CLI was mostly a remainder from when the language server only supported Maven and is not consistent with the other `ClassPathResolver`s.

I would suggest to remove this snippet. If needed, we could log them on a higher level instead (e.g. once all `ClassPathResolver`s have completed resolution).